summoner-cli/summoner.cabal

```diff
@@ -153,17 +159,21 @@ library
                      , validation-selective ^>= 0.1.0.0
 
 executable summon
-  import:              common-options
+  import:              common-options, exe-options
   hs-source-dirs:      app
   main-is:             Cli.hs
   build-depends:       summoner
 
-  ghc-options:         -threaded
-                       -rtsopts
-                       -with-rtsopts=-N
+executable gg
+  import:              exe-options
+  hs-source-dirs:      golden
+  main-is:             Main.hs
+  build-depends:       base
```

Comment:
I wasn't able to use `common-options` here because GHC outputs warning that `base` is not used in this executable (due to `-Wunused-packages`) 😒 So I don't use `relude` and warnings in this simple small file, just `base`, no ketchup, raw `base`